docs/community/pr-velocity.md

```diff
@@ -1,11 +1,11 @@
 ---
 id: pr-velocity
 title: 'PR Velocity'
-sidebar_label: 'PR Velocity'
+sidebar_label: PR Velocity
 keywords:
-  - pr-velocity 
+  - pr-velocity
 ---
 
-The PR Velocity as shown in purple in the screenshot below, measures the percentage and amount of days that  pull requests that are merged. It's a great way to track the progress of your open source contributions.
+The PR Velocity as shown in purple in the screenshot below, measures the percentage and amount of days that pull requests that are merged. It's a great way to track the progress of your open source contributions.
 
 ![PR Velocity Example](../../static/img/pr-velocity-example.png)
```

Comment:
We will have a PR Velocity in the Glossary. Should we move this part there or somewhere else? Because I don't think this file belongs in `community` section.